BraveRewardsUI/Extensions/BraveLedgerExtensions.swift

```diff
@@ -76,3 +76,12 @@ extension BraveLedger {
     }
   }
 }
+
+struct BATTipOption: DisplayableOption {
```

Comment:
This isn't a ledger extension, since its really only used for the UI selection, it can go with the the refactored DetailActionableRow

Reply:
Yeah, kept it here for suggestions :)

Reply:
Removed this in favour of BATValue